### DISCUSSION TITLE
Comments behind a version are removed from gh actions

### DISCUSSION THREAD

**Discussion by nils-a** [renovatebot/renovate#9462] [Request Help]
e.g. see https://github.com/nils-org/Topshelf.LightCore/pull/43/files.

while I personally do not mind the change, I can not help but wonder: Is it a bug or a feature? 😉

**Comment by viceice** [comment]
I would say it's a bug, but hard to fix. I think we use a yaml parser wich removed those comments when updating the value.

**Reply by nils-a** [reply]
From a "user" perspective I'd say it's bug, too. (However easy to work around...)

**Reply by rarkins** [reply]
Agreed. Please create a minimal reproduction repo (i.e. repo with just this one example action) and a bug report issue.

**Reply by nils-a** [reply]
https://github.com/renovatebot/renovate/issues/9473
